tests/wicked/startandstop/sut/t18_firewall_extension.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: wicked
+# Summary: Verify firewalld zone settings after ifup/ifdown/ifreload
+# Maintainer:
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+use Utils::Systemd;
+use version_utils 'is_sle';
+
+has wicked_version => '>=0.6.70';
+
+sub run {
+    my ($self, $ctx) = @_;
+    my $ifc = $ctx->iface();
+
+    return if ($self->skip_by_wicked_version());
+
+    # No firewalld on sles 12-SP5 (bsc#1180116)
+    if (is_sle('<=12-SP5')) {
```

Comment:
```suggestion
    if (is_sle('<12-SP5')) {
```
otherwise SLE 12-SP5 will be in

Reply:
Thx!

Reply:
Ah but `is_sle('<=12-SP5')` is what we want here, cause we will skipp everything lower and including `12-SP5`.

Reply:
What we could do is
```perl
if (!is_sle('>12-SP5')) {
```

Reply:
ah right , my bad